src/common/stattextcmn.cpp

```diff
@@ -213,8 +213,46 @@ class wxLabelWrapper : public wxTextWrapper
 
 void wxStaticTextBase::Wrap(int width)
 {
+    if (width == m_currentWrap) return;
```

Comment:
Please don't do this:

```suggestion
    if (width == m_currentWrap)
        return;
```